src/extension/options-page/index.tsx

```diff
@@ -0,0 +1,180 @@
+import '../../provider.worker'
+
+import React, { useRef, useMemo, useContext, useEffect, useState } from 'react'
+import { CssBaseline, useMediaQuery } from '@material-ui/core'
+import { ThemeProvider, makeStyles, createStyles } from '@material-ui/core/styles'
+
+import PeopleOutlinedIcon from '@material-ui/icons/PeopleOutlined'
+import CreditCardIcon from '@material-ui/icons/CreditCard'
+import BookmarkBorderOutlinedIcon from '@material-ui/icons/BookmarkBorderOutlined'
+import SettingsOutlinedIcon from '@material-ui/icons/SettingsOutlined'
+
+import { HashRouter as Router, Route, Switch, Redirect } from 'react-router-dom'
+
+import { MaskbookDarkTheme, MaskbookLightTheme } from '../../utils/theme'
+
+import '../../setup.ui'
+import { SSRRenderer } from '../../utils/SSRRenderer'
+
+import { SnackbarProvider } from 'notistack'
+
+import { I18nextProvider } from 'react-i18next'
+import ResponsiveDrawer from './Drawer'
+
+import { DialogRouter } from './DashboardDialogs/DialogBase'
+import DashboardInitializeDialog from './Initialize'
+import { useI18N } from '../../utils/i18n-next-ui'
+import i18nNextInstance from '../../utils/i18n-next'
+import FooterLine from './DashboardComponents/FooterLine'
+
+import DashboardPersonasRouter from './Router/Personas'
+import DashboardWalletsRouter from './Router/Wallets'
+import DashboardContactsRouter from './Router/Contacts'
+import DashboardSettingsRouter from './Router/Settings'
+
+const OptionsPageRouters = (
+    <>
+        <Switch>
+            <Route path="/personas/" component={DashboardPersonasRouter} />
+            <Route path="/wallets/" component={DashboardWalletsRouter} />
+            <Route path="/contacts/" component={DashboardContactsRouter} />
+            <Route path="/settings/" component={DashboardSettingsRouter} />
+            <DialogRouter path="/initialize" component={DashboardInitializeDialog} onExit={'/'} fullscreen />
+            <Redirect path="*" to="/home/" />
+        </Switch>
+    </>
+)
+
+function DashboardWithProvider() {
+    const isDarkTheme = useMediaQuery('(prefers-color-scheme: dark)')
+    return (
+        <I18nextProvider i18n={i18nNextInstance}>
+            <ThemeProvider theme={isDarkTheme ? MaskbookDarkTheme : MaskbookLightTheme}>
+                <SnackbarProvider
+                    maxSnack={30}
+                    anchorOrigin={{
+                        vertical: 'top',
+                        horizontal: 'right',
+                    }}>
+                    <Router>
+                        <CssBaseline />
+                        <Dashboard></Dashboard>
+                    </Router>
+                </SnackbarProvider>
+            </ThemeProvider>
+        </I18nextProvider>
+    )
+}
+
+const useStyles = makeStyles(theme =>
+    createStyles({
+        wrapper: {
+            '--monospace': 'SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace',
+
+            '--primary': theme.palette.primary.main,
```

Comment:
strange... why do you want to use css variable when the value is directly inherit from the theme?